### DISCUSSION TITLE
bitwarden_rs on Azure WebApp

### DISCUSSION THREAD

**Discussion by martytcs** [dani-garcia/vaultwarden#1609] [Q&A] [answered]
Hi All,  hoping there are some experts here that could give me some guidance.

I tried setting up bitwarden_rs on azure webapp with a little success
![image](https://user-images.githubusercontent.com/82550944/114796553-a1510080-9dd4-11eb-8108-b2d09d2c4e1b.png)

the site loads up but when I started using it and reboot, I lose all the settings/users/passwords.
reading up more, I think it is because I need to put the data file on persistent storage.

I then create an azurefile and attached it as /bw-data
![image](https://user-images.githubusercontent.com/82550944/114796732-fbea5c80-9dd4-11eb-9547-b2675454dd40.png)

set the environmental variable DATA_FOLDER = bw-data
![image](https://user-images.githubusercontent.com/82550944/114797482-9eefa600-9dd6-11eb-9855-b292b74f9003.png)

when i power the webapp up,  it does create all the files/folder to the attached volume
![image](https://user-images.githubusercontent.com/82550944/114797325-43251d00-9dd6-11eb-84f9-36030ad57f0f.png)

expecting this to work, navigated to the url and got an error page instead.
![image](https://user-images.githubusercontent.com/82550944/114796884-51266e00-9dd5-11eb-8fa6-f3433c87df78.png)

I'm not sure if I am missing some steps.  has anyone done something similar before?

**Comment by RealOrangeOne** [comment, marked as answer]
I think you're seeing the same issue as https://github.com/dani-garcia/bitwarden_rs/discussions/1581. See response there.

**Comment by mazrafgit** [comment]
@martytcs did You resolve this issue?
